src/components/UserActionFeedback/stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { UserActionFeedback } from ".";
+
+const DEFAULT_PROPS = {
+  page: "message_sent",
```

Comment:
"message_sent" should be defined in an `enum`

Reply:
Done